steps/cadence-genus-genlib/scripts/set_libs.tcl

```diff
@@ -6,8 +6,7 @@
 # Author : Stephen Richardson
 # Date   : December 2020
 
-# Note similar code in nearby scripts
-# FIXME maybe they should all share a common code base
+# FIXME Note similar code in nearby scripts, maybe they should all share a common code base
```

Comment:
Let's remove this comment. We are watching this problem already in our dev talks.

Reply:
done. also removed similar comment from designer-interface.tcl